module/scheduler_components/commands.cpp

```diff
@@ -40,6 +40,10 @@ void KeyValueCommands::Execute(Transaction& txn) {
       if (read_set.contains(src) && write_set.contains(dst)) {
         write_set[dst] = read_set.at(src);
       }
+    } else if (cmd_ == "ABORT") {
+      if (write_set.contains(args_[0]) || read_set.contains(args_[0])) {
```

Comment:
Nope. Previously, I added the key so that we could "control" which partition to abort but then I realized all partitions ran the same code. Actually this "ABORT" command is not needed either. I can just simply have an `else` here and abort for any token that we don't understand.

Reply:
ABORT <key> <value> would kind of make sense to me. We could abort if the key != value, so we'd have primitive logical aborts. On the other hand it doesn't affect performance, so maybe we should just keep things simple.